docs/ops/convolution/ConvolutionBackpropData_1.md

```diff
@@ -132,3 +144,67 @@ else:
     </output>
 </layer>
 ```
+
+*Example 2: 2D ConvolutionBackpropData with output_padding*
+
+```xml
+<layer id="5" name="upsampling_node" type="ConvolutionBackpropData">
+    <data dilations="1,1" pads_begin="0,0" pads_end="0,0" strides="3,3" output_padding="2,2" auto_pad="explicit"/>
+    <input>
+        <port id="0">
+            <dim>1</dim>
+            <dim>20</dim>
+            <dim>2</dim>
+            <dim>2</dim>
+        </port>
+        <port id="1">
+            <dim>20</dim>
+            <dim>10</dim>
+            <dim>3</dim>
+            <dim>3</dim>
+        </port>
+    </input>
+    <output>
+        <port id="0" precision="FP32">
+            <dim>1</dim>
+            <dim>10</dim>
+            <dim>8</dim>
+            <dim>8</dim>
+        </port>
+    </output>
+</layer>
+```
+
+*Example 3: 2D ConvolutionBackpropData with output_shape input*
+
+```xml
+<layer id="5" name="upsampling_node" type="ConvolutionBackpropData">
+    <data dilations="1,1" pads_begin="1,1" pads_end="1,1" strides="1,1" output_padding="0,0" auto_pad="valid"/>
+    <input>
+        <port id="0">
+            <dim>1</dim>
+            <dim>20</dim>
+            <dim>224</dim>
+            <dim>224</dim>
+        </port>
+        <port id="1">
+            <dim>20</dim>
+            <dim>10</dim>
+            <dim>3</dim>
+            <dim>3</dim>
+        </port>
+        <port id="2">
+            <dim>450</dim>
+            <dim>450</dim>
```

Comment:
Wrong example. The shape of the third input should be `[2]` and the **elements** in this tensor should be `[450, 450]`

Reply:
Corrected.